kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/api/UploadBackendClient.kt

```diff
@@ -50,37 +50,37 @@ open class UploadBackendClient(
         uploadBackendBaseUrl = uploadBackendBaseUrl.trimEnd('/')
     }
 
-    fun pollRecords(configuration: PollDTO): RecordContainerDTO = httpClient.executeRequest {
+    open fun pollRecords(configuration: PollDTO): RecordContainerDTO = httpClient.executeRequest {
```

Comment:
why do we make the functions `open`? do we have any class extending this client? or for any other purpose?

Reply:
Yes, for purpose of mocking them in tests